content-src/components/DiscoveryStreamComponents/CardGrid/_CardGrid.scss

```diff
@@ -68,7 +68,7 @@ $col4-header-font-size: 14;
       }
 
       .excerpt {
-        @include limit-visibile-lines(3, 24, 15);
+        @include limit-visibile-lines(4, 24, 15);
```

Comment:
This is just going to change excerpt length to 4 only for 9, 10, 11, 12 column card grid layout. Is that the ask?

To change it everywhere for DSCard including card list in Hero, we should be [changing here](https://github.com/mozilla/activity-stream/blob/master/content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss#L107)

Reply:
Ah I see.